controllers/Auth.js

```diff
@@ -234,8 +130,11 @@ exports.signUp = async (req, res) => {
       //Hash password
       const hashedPassword = await bcrypt.hash(password, 10);
 
-      //entry create in DB
+      //Create the user
+      let approved ="";
+      approved === "Instructor" ? (approved = false) : (approved = true);
```

Comment:
if we are setting approved = "", why are we checking if it's Instructor

Reply:
will that code really run anytime

Reply:
missing code will be coded later on, coz the approved variable is static now, once we fetch it from req.body it will be used. We will make it dunamic once testing is started

Reply:
ohh okay okay